feature-utils/poly-import/src/importer.js

```diff
@@ -48,52 +47,92 @@ class ImporterExecutionResult {
     }
 }
 
-export async function runImporter(importerClass, zipFile, facebookAccount) {
+export async function runImporter({ importerClass, zipFile, pod, account }) {
     const importer = new importerClass();
 
     const telemetry = new Telemetry();
 
     try {
-        const status = await importer.import({
+        const response = await importer.import({
             zipFile,
-            facebookAccount,
+            pod,
+            //this is to support old importers without needing to change all at once
+            //TODO: change all importers so this can go
+            account,
+            facebookAccount: account,
         });
-        return new ImporterExecutionResult(
-            importer,
-            status,
-            telemetry.elapsedTime()
-        );
+        //Currently we have to do this check as not all importers return a result even when
+        //executing successfully. We can go back to destructuring after all importers have been changed
+        return {
+            report: new ImporterExecutionReport({
```

Comment:
Result did not exist before and instead of adding directly to facebookAccount importers return the data as _result_. I specifically wanted to have a less oop forced approach here as it's really not necessary IMHO

Reply:
it's more about settling for a certain interface. That way, there's a single source of truth for the attribute names.